### PR TITLE
swap shas for images to fix update issues

### PR DESCRIPTION
**Pull request by JasonPowr**
Quick fix to change the sha's for the image versions to the tags

### CODE REVIEW

File: examples/values-ez.yaml [hunk 1]
```diff
@@ -226,7 +226,7 @@ scaffold:
     deployment:
       registry: quay.io
       repository: securesign/tuf/server
-      version: sha256:b1cbb379f5c4b315bc5a7d7a37865ab61669a0a07002b6920374ebae3b37699c
+      version: d2e1b58
```
Comment (sallyom) [inline examples/values-ez.yaml:229]: that's a weird tag, but it is there, i see it :shrug:

File: examples/values-ez.yaml [hunk 2]
```diff
@@ -166,24 +166,24 @@ scaffold:
       image:
         registry: quay.io
         repository: securesign/trillian_log_signer
-        version: sha256:ba8d467c38c1eb6a24f843b855e6b01fd5b524d7db45a82b4092a681b561c143
+        version: 	v1.2.2
```
Comment (sallyom) [inline examples/values-ez.yaml:169]:
```suggestion
        version: v1.2.2
```

File: examples/values-ez.yaml [hunk 3]
```diff
@@ -67,13 +67,13 @@ scaffold:
...
         repository: securesign/createcerts
-        version: sha256:cc10d4f86a893de36e6a269558612b1343dc64c359b927cfcc77ba617eab8283
+        version: 	v0.6.4
         pullPolicy: IfNotPresent
     server:
       image:
         registry: quay.io
         repository: securesign/fulcio
-        version: sha256:9239e56419073d57bba2ec217c0cf64499b7d4e686a875394d34a802ad99fbad
+        version: 	v1.4.0
```
Comment (sallyom) [inline examples/values-ez.yaml:76]:
```suggestion
        version: v1.4.0
```

File: examples/values-ez.yaml [hunk 4]
```diff
@@ -67,13 +67,13 @@ scaffold:
       image:
         registry: quay.io
         repository: securesign/createcerts
-        version: sha256:cc10d4f86a893de36e6a269558612b1343dc64c359b927cfcc77ba617eab8283
+        version: 	v0.6.4
```
Comment (sallyom) [inline examples/values-ez.yaml:70]:
```suggestion
        version: v0.6.4
```

File: examples/values-ez.yaml (hunk 1 above)
Comment (sallyom) [reply examples/values-ez.yaml:229]:
```suggestion
      version: latest
```

File: examples/values-sigstore-openshift.yaml [hunk 5]
```diff
@@ -115,13 +115,13 @@ scaffold:
       image:
         registry: quay.io
         repository: securesign/createtree
-        version: sha256:c03cdc797e8d248bfa7371b34b1eda582702e5386fdbb543e5a121afae255fb7
+        version: 	v0.6.4
```
Comment (sallyom) [inline examples/values-sigstore-openshift.yaml:118]:
```suggestion
        version: v0.6.4
```

File: examples/values-sigstore-openshift.yaml [hunk 6]
```diff
@@ -141,15 +141,15 @@ scaffold:
     deployment:
       registry: quay.io
       repository: securesign/tuf/server
-      version: sha256:b1cbb379f5c4b315bc5a7d7a37865ab61669a0a07002b6920374ebae3b37699c
+      version: d2e1b58
```
Comment (sallyom) [inline examples/values-sigstore-openshift.yaml:144]:
```suggestion
      version: latest
```

**Review by sallyom — CHANGES_REQUESTED** [review]
added suggestions for spacing

**Comment by JasonPowr** [comment]
Apologies for spacing issue, should be all good now :+1:

**Pull request merged**